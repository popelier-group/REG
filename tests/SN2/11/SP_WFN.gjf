# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.70293000   -0.00005900    0.00021000
 H                 -0.85655900   -0.90277600   -0.56231100
 H                 -0.85656900    0.93847100   -0.50026900
 H                 -0.85648300   -0.03588500    1.06326400
 Br                 1.60299800    0.00000400   -0.00001400
 Cl                -3.30239700    0.00001700   -0.00006000

frame.wfn
