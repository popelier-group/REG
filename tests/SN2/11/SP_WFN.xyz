
6

C        0.004456    0.052240    0.588748
H        0.645938    0.911364    0.661793
H        0.421007   -0.909852    0.824945
H       -1.050001    0.195893    0.738513
Br       -0.013480   -0.151350   -1.708105
Cl        0.025183    0.281553    3.177997
