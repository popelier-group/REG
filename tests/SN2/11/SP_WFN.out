 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.11/Gau-138780.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.11/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_haswell/g16/l1.exe PID=    138782.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.70293  -0.00006   0.00021 
 H                    -0.85656  -0.90278  -0.56231 
 H                    -0.85657   0.93847  -0.50027 
 H                    -0.85648  -0.03589   1.06326 
 Br                    1.603     0.       -0.00001 
 Cl                   -3.3024    0.00002  -0.00006 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.702930   -0.000059    0.000210
      2          1           0       -0.856559   -0.902776   -0.562311
      3          1           0       -0.856569    0.938471   -0.500269
      4          1           0       -0.856483   -0.035885    1.063264
      5         35           0        1.602998    0.000004   -0.000014
      6         17           0       -3.302397    0.000017   -0.000060
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.074677   0.000000
     3  H    1.074673   1.842292   0.000000
     4  H    1.074684   1.842280   1.842283   0.000000
     5  Br   2.305928   2.679666   2.679635   2.679719   0.000000
     6  Cl   2.599467   2.667074   2.667017   2.667291   4.905395
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.004456    0.052240    0.588748
      2          1           0        0.645938    0.911364    0.661793
      3          1           0        0.421007   -0.909852    0.824945
      4          1           0       -1.050001    0.195893    0.738513
      5         35           0       -0.013480   -0.151350   -1.708105
      6         17           0        0.025183    0.281553    3.177997
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         147.7469244           0.8507910           0.8507909
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       173.7218145613 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  7.69D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147513.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.38892071     A.U. after   11 cycles
            NFock= 11  Conv=0.10D-07     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.64678-101.28992 -62.29971 -56.11903 -56.11668
 Alpha  occ. eigenvalues --  -56.11668 -10.08687  -9.20326  -8.50863  -6.96304
 Alpha  occ. eigenvalues --   -6.96072  -6.96072  -6.33651  -6.32884  -6.32884
 Alpha  occ. eigenvalues --   -2.44792  -2.44582  -2.44582  -2.43976  -2.43976
 Alpha  occ. eigenvalues --   -0.60910  -0.55659  -0.52288  -0.30399  -0.30398
 Alpha  occ. eigenvalues --   -0.18927  -0.12726  -0.12726  -0.08561  -0.08561
 Alpha  occ. eigenvalues --   -0.08489
 Alpha virt. eigenvalues --    0.09282   0.11359   0.14860   0.14861   0.15553
 Alpha virt. eigenvalues --    0.17833   0.18250   0.18251   0.20562   0.21841
 Alpha virt. eigenvalues --    0.21842   0.23614   0.23614   0.24782   0.31027
 Alpha virt. eigenvalues --    0.31693   0.31694   0.35834   0.39531   0.52109
 Alpha virt. eigenvalues --    0.54097   0.54097   0.55901   0.55901   0.62563
 Alpha virt. eigenvalues --    0.65672   0.65673   0.70296   0.75417   0.76004
 Alpha virt. eigenvalues --    0.76004   0.80522   0.80522   0.83769   0.83770
 Alpha virt. eigenvalues --    0.85480   0.90961   1.11772   1.11773   1.12065
 Alpha virt. eigenvalues --    1.12119   1.12120   1.15303   1.22514   1.22514
 Alpha virt. eigenvalues --    1.33154   1.53600   1.63532   1.63533   1.70594
 Alpha virt. eigenvalues --    1.94466   1.94467   1.99576   2.14018   2.14018
 Alpha virt. eigenvalues --    2.22181   2.22182   2.29966   2.49785   2.49788
 Alpha virt. eigenvalues --    2.57679   2.62608   2.62608   2.65724   2.75683
 Alpha virt. eigenvalues --    2.75684   3.01535   3.22045   3.85615   3.85616
 Alpha virt. eigenvalues --    4.31463   4.31463   4.35794   4.35794   4.56882
 Alpha virt. eigenvalues --    6.74322   7.72478   7.72479   7.82138   9.87480
 Alpha virt. eigenvalues --   24.04079  26.06043  26.06046  26.25819  48.04308
 Alpha virt. eigenvalues --  215.84789 289.92689 289.92689 290.023401020.81560
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.035934   0.416488   0.416494   0.416468   0.066034  -0.052775
     2  H    0.416488   0.518250  -0.032463  -0.032470  -0.043046  -0.008596
     3  H    0.416494  -0.032463   0.518246  -0.032468  -0.043051  -0.008599
     4  H    0.416468  -0.032470  -0.032468   0.518265  -0.043033  -0.008589
     5  Br   0.066034  -0.043046  -0.043051  -0.043033  35.565831   0.006673
     6  Cl  -0.052775  -0.008596  -0.008599  -0.008589   0.006673  17.809334
 Mulliken charges:
               1
     1  C   -0.298644
     2  H    0.181837
     3  H    0.181840
     4  H    0.181825
     5  Br  -0.509409
     6  Cl  -0.737449
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.246858
     5  Br  -0.509409
     6  Cl  -0.737449
 Electronic spatial extent (au):  <R**2>=           1143.8717
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             -0.0527    Y=             -0.5844    Z=             -6.5985  Tot=              6.6246
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -43.7839   YY=            -44.1489   ZZ=            -90.6749
   XY=             -0.0329   XZ=             -0.3716   YZ=             -4.1545
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             15.7520   YY=             15.3870   ZZ=            -31.1390
   XY=             -0.0329   XZ=             -0.3716   YZ=             -4.1545
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             -1.4194  YYY=             -5.4507  ZZZ=           -176.2993  XYY=              0.8088
  XXY=             -1.2923  XXZ=            -18.8272  XZZ=             -1.0898  YZZ=            -12.2911
  YYZ=            -19.7100  XYZ=             -0.1725
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -70.4552 YYYY=            -81.3080 ZZZZ=          -1825.4911 XXXY=             -0.5393
 XXXZ=             -6.1551 YYYX=             -0.3063 YYYZ=            -61.7152 ZZZX=             -8.4086
 ZZZY=            -94.3095 XXYY=            -25.2138 XXZZ=           -253.7607 YYZZ=           -258.3916
 XXYZ=            -20.1388 YYXZ=             -1.1721 ZZXY=             -0.5485
 N-N= 1.737218145613D+02 E-N=-7.699641172857D+03  KE= 3.069964706180D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE773\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-0.70293,-0.000059,0.00021\H,0,-0.856559,-0.902776,-0.562311\H,0,-
 0.856569,0.938471,-0.500269\H,0,-0.856483,-0.035885,1.063264\Br,0,1.60
 2998,0.000004,-0.000014\Cl,0,-3.302397,0.000017,-0.00006\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.3889207\RMSD=9.956e-09\Dipole=2.606314
 9,-0.0000591,0.0002117\Quadrupole=-23.4269005,11.7134934,11.7134071,0.
 0002229,-0.0007512,-0.0000056\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 THE MORE POWERFUL THE METHOD, THE MORE CATASTROPHIC THE ERRORS.
     -- M.D. KAMEN
 Job cpu time:       0 days  0 hours  0 minutes 17.8 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 10.4 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:04 2020.
